.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+###### Description
+
+
+###### System Info
+macOS 10.
+Xcode 8.
+
+###### Verification
+- [ ] Have you checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
+- [ ] Have you referenced relating tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
```

Comment:
There is already a wiki page dealing with [formatting of commit messages](https://trac.macports.org/wiki/CommitMessages). It's linked in our WorkingWithGit wiki page, but the path to there is not easy to find - even if you know where to search for it.